person.rb

```diff
@@ -7,15 +10,22 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
 
   def of_age?
     @age >= 18
   end
 
+  def correct_name
+    @name
+  end
+
   private :of_age?
 
   def can_use_services?
     of_age?
   end
+
+  has_many :rental
```

Comment:
- [ ] I see you're using active record associations in a plain ruby app. Please note that this will throw an error especially since there's not a Gemfile that includes the ActiveRecord gem.
If you want to use ActiveRecord in a regular Ruby app. Please follow this [guide](https://www.techcareerbooster.com/blog/use-activerecord-in-your-ruby-project#:~:text=ActiveRecord%20is%20a%20gem%20that,RDBMS%2C%20like%20MySQL%20or%20PostgreSQL.).